src/speculators/convert/eagle/cli.py

```diff
@@ -0,0 +1,95 @@
+"""
```

Comment:
Remove in favor of Python API entrypoints which are imported in root __main__.py file